tests/cephadm/test_cephadm_b2b_upgrade.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
Can this module be generic one and the configuration can dictate if the process is build to build or release to release?